lib/Dialect/EmitC/Conversion/MHLOToEmitC.cpp

```diff
@@ -41,6 +41,7 @@ SmallVector<Attribute, 2> indexSequence(int64_t n, MLIRContext *ctx) {
       }));
 }
 
+/// Convert `mhlo.constant` into an `emic.const` operation.
```

Comment:
This is a copy pasted typo, search for `emic` in your changes, it appears in multiple places

```suggestion
/// Convert `mhlo.constant` into an `emitc.const` operation.
```

Reply:
Good catch! All done :)